recipe/meta.yaml

```diff
@@ -44,3 +44,4 @@ about:
 extra:
   recipe-maintainers:
     - mariusvniekerk
+    - ericdill
```

Comment:
Yup. I'll post on that issue giving Marius the green light to add me to whatever he sees fit